forms/migrations/0022_create_form_view.sql

```diff
@@ -0,0 +1,75 @@
+DROP VIEW IF EXISTS entry_view;
+CREATE OR REPLACE VIEW entry_view AS
+SELECT
```

Comment:
I tried to run this query locally, but I did not get any results. Should I expect to or am I likely lacking some specific kind of data?

Reply:
Do you have entries?

Reply:
Well, There is something weird

Reply:
I found the issue
`JOIN leasing_lease ON plotsearch_plotsearchtarget.reservation_identifier_id = leasing_lease.id` I don't have any of those. So are these representing all of the leases that happened? In that case I think the view should be renamed to be more descriptive, and perhaps in any case it would be useful, and to add further context maybe a comment of what the view is for

Reply:
Those are just added info. But I suppose I have to find out how to make some if condition there

Reply:
Is that sort of optional? What about outer join?

Reply:
This should be fixed now